internal/attestation/crafter/crafter_test.go

```diff
@@ -209,6 +220,24 @@ func (s *crafterSuite) TestLoadSchema() {
 			contractPath: "testdata/contracts/invalid.yaml",
 			wantErr:      true,
 		},
+		{
+			name:         "policies",
+			contractPath: "testdata/contracts/with_policy_embedded.yaml",
+		},
+		{
+			name:         "missing policy",
+			contractPath: "testdata/contracts/with_missing_policy.yaml",
+			wantErr:      true,
+		},
+		{
+			name:         "missing script",
+			contractPath: "testdata/contracts/with_policy_missing_rego.yaml",
+			wantErr:      true,
+		},
+		{
+			name:         "rego policy",
+			contractPath: "testdata/contracts/with_rego.yaml",
```

Comment:
should this test somehow check the result? `want`?

Reply:
Done!